src/test/java/com/puppycrawl/tools/checkstyle/filters/SuppressionPatchFilterTest.java

```diff
@@ -451,10 +473,17 @@ private void testByConfig(String configPath, String inputFile, String[] expected
             final List<String> actuals = lnr.lines().limit(expected.length)
                     .sorted().collect(Collectors.toList());
             Arrays.sort(expected);
-
-            for (int i = 0; i < expected.length; i++) {
-                final String expectedResult = path + ":" + expected[i];
-                assertEquals("error message " + i, expectedResult, actuals.get(i));
+            if (file.isDirectory()) {
+                for (int i = 0; i < expected.length; i++) {
+                    final String expectedResult = path + "/" + expected[i];
+                    assertEquals("error message " + i, expectedResult, actuals.get(i));
+                }
+            }
+            else {
+                for (int i = 0; i < expected.length; i++) {
+                    final String expectedResult = path + ":" + expected[i];
+                    assertEquals("error message " + i, expectedResult, actuals.get(i));
+                }
```

Comment:
One char difference comparing to the above for loop. Please avoid duplication.

Reply:
done